doc/nrf/protocols/lte/index.rst

```diff
@@ -0,0 +1,215 @@
+The nRF91 Series supports software SIM, which allows the usage of software SIM-based solutions to reduce energy consumption associated with physical SIMs.
+The following are some of the key advantages of using software SIM:
+
+* Cost savings and simplified design by excluding the SIM hardware layer.
+* Power savings, especially in idle mode using eDRX.
+
+See the software SIM support section of the `Cellular IoT unique features`_ documentation for information on software SIM.
+
+The software SIM support is available in the |NCS| from the v2.5.0 release.
+The `Onomondo SoftSIM integration with the nRF91 Series`_ guide describes the integration of software SIM into |NCS|, and the :ref:`nrf_modem_softsim` guide demonstrates the transfer of SIM data between the modem and the application.
+
+Power consumption
+=================
+
+LTE-M and NB-IoT both support Power Saving Mode (PSM) or Extended Discontinuous Reception (eDRX) to minimize power consumption.
+For more information on power-saving techniques, refer to the DevAcademy's `Power saving techniques`_ documentation.
+
+Security
+========
+
+The nRF91 Series devices include a range of security features, such as Arm TrustZone and Arm CryptoCell for secure application and data handling.
+Security best practices are also implemented to protect data transmitted over the LTE network.
+This also includes using Transport Layer Security (TLS) for Transmission Control Protocol (TCP) and Datagram Transport Layer Security (DTLS) for User Datagram Protocol (UDP) as security protocols.
+For more information, refer to the `Security protocol for cellular IoT`_ documentation.
+
+Getting started with cellular products
+**************************************
+
+For getting started with the cellular products, you can refer to the :ref:`ug_nrf91` documentation, which guides through the features, getting started steps, and advanced usage of the boards.
+For a self-paced, hands-on course, enroll in DevAcademy's `Cellular IoT Fundamentals course`_ , which describes the LTE and NB-IoT technologies and the software architecture of a cellular IoT application based on Nordic Semiconductor’s nRF91 Series device.
+
+The :ref:`nrf9160_ug_network_mode` section provides information about switching between LTE-M and NB-IoT.
+
+Cellular IoT in |NCS|
+*********************
+
+The following sections explain the different applications, libraries, and samples that use the cellular IoT products.
+
+Modem library
+=============
+
+.. include:: ../../device_guides/working_with_nrf/nrf91/nrf91_features.rst
+    :start-after: nrf91_modem_lib_start
+    :end-before: nrf91_modem_lib_end
+
+Library support
+===============
+
+The following |NCS| libraries are used for cellular IoT:
+
+* :ref:`lib_modem`
+* :ref:`lib_networking`
+* :ref:`liblwm2m_carrier_readme` library
+* :ref:`mod_memfault` library
+
+Applications and samples
+========================
+
+The following applications use the cellular IoT in |NCS|:
+
+* :ref:`asset_tracker_v2`
```

Comment:
Add the https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/cellular/nrf_cloud_multi_service/README.html sample if it is not already in `cellular_samples`.

Reply:
Yeah, the sample is in the cellular_samples folder.